doc/source/security/data-masking.rst

```diff
@@ -0,0 +1,191 @@
+.. _data-masking:
+
+==================================================================
+Data Masking
+==================================================================
+
+This feature is **Experimental** quality.
+
+This feature was implemented in |Percona Server| version :rn:`5.7.31-34`.
```

Comment:
This will not be in 5.7.31-34 but next series 5.7.32 or maybe an interim release of 5.7.31-35

Reply:
Added a placeholder.